Add workflows to auto-generate project files (#208)
Azure/bicep-registry-modules: CONTRIBUTING.md

Once the module files are validated locally, you can commit your changes and open a pull request. You must link the new module proposal in the pull request description if you are trying to add a new module. Adding or updating multiple modules is not supported and will cause a failure in the pull request validation CI, so please only add or change one module at a time.

### Optional: Enable Auto Generation with GitHub Actions

> Enable optional GitHub Worksflows in your fork to enable auto-generation of assets with our [GitHub Action](/.github/workflows/push-auto-generate.yml).

Author: ```suggestion
Enable optional GitHub Workflows in your fork to enable auto-generation of assets with our [GitHub Action](/.github/workflows/push-auto-generate.yml).
```

> To trigger GitHub Actions after auto-generation, [add a GitHub PAT](https://stackoverflow.com/a/69063453/1341806) as a secret in your forked repository called `PAT`.

Author: ```suggestion
To trigger GitHub Actions after auto-generation, [add a GitHub PAT](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token) as a secret in your forked repository called `PAT`.
```

> To trigger GitHub Actions after auto-generation, [add a GitHub PAT](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token) as a secret in your forked repository called `PAT`.

Author: ```suggestion
In order to trigger GitHub Actions after auto-generation, [add a GitHub PAT](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token) as a secret in your forked repository called `PAT`.
```

> This `PAT` should NOT inclucde the `workflow` scope.

Author: ```suggestion
This `PAT` should NOT include the `workflow` scope.
```

Once the module files are validated locally, you can commit your changes and open a pull request. You must link the new module proposal in the pull request description if you are trying to add a new module. Adding or updating multiple modules is not supported and will cause a failure in the pull request validation CI, so please only add or change one module at a time.

> ### Enable Auto Generation with GitHub Actions

Reviewer: Can we mention that the step is optional?

Enable GitHub Worksflows in your fork to enable auto-generation of assets with our [GitHub Action](/.github/workflows/push-auto-generate.yml).
To trigger GitHub Actions after auto-generation, [add a GitHub PAT](https://stackoverflow.com/a/69063453/1341806) as a secret in your forked repository called `PAT`.
This `PAT` should NOT inclucde the `workflow` scope.

> To autoformat files in the `.github` directory, an dedicated token named `WORKSPACE_PAT` should be used for this [GitHub Action](/.github/workflows/push-workflow-format.yml).
> 

Author: ```suggestion
```
